Sources/MongoSwift/MongoClient.swift

```diff
@@ -4,6 +4,9 @@ import NIOConcurrencyHelpers
 
 /// Options to use when creating a `MongoClient`.
 public struct ClientOptions: CodingStrategyProvider {
+    /// Authentication Credential
```

Comment:
I think this docstring could be more descriptive, something like "Specifies authentication options for use with the client.